### PR TITLE
avoid trailing underscore

### PR DESCRIPTION
**Pull request by jetersen**
as it is not considered a canonical release

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: git-changelist-maven-extension/src/main/java/io/jenkins/tools/incrementals/git_changelist_maven_extension/Main.java [hunk 1]
```diff
@@ -36,6 +36,7 @@
 import java.util.Properties;
 import java.util.Set;
 import java.util.TreeSet;
+import org.apache.commons.lang3.StringUtils;
```
Comment (jetersen) [inline git-changelist-maven-extension/src/main/java/io/jenkins/tools/incrementals/git_changelist_maven_extension/Main.java:39]:
```suggestion
```

**Review by basil — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Review by jglick — APPROVED** [review]

**Comment by jetersen** [comment]
Well that was easy to land 😆

**Comment by jglick** [comment]
https://github.com/jenkinsci/incrementals-tools/releases/tag/parent-1.4

**Comment by jglick** [comment]
Yeah, well, MRP.

**Comment by jetersen** [comment]
@jglick fixable via config/property: https://github.com/jenkinsci/configuration-as-code-plugin/blob/bf985b894e408c24b07902a43a1d228758e5ce57/pom.xml#L25

**Comment by jglick** [comment]
I know, just do not much care. If you do, fix it w/ https://github.com/jenkinsci/incrementals-tools/blob/f909d7b7652aaf2cdc660cdf7fc3852f0eeada40/.github/release-drafter.yml#L2

**Comment by basil** [comment]
Caused https://github.com/jenkinsci/incrementals-tools/issues/34.

**Pull request merged**